deploy/olm-catalog/nxiq-operator-certified/manifests/nxiq-operator-certified.clusterserviceversion.yaml

```diff
@@ -0,0 +1,340 @@
+            "name": "example-nexusiq"
+          },
+          "spec": {
+            "deployment": {
+              "postStart": {
+                "command": null
+              },
+              "preStart": {
+                "command": null
+              }
+            },
+            "fullnameOverride": "",
+            "imagePullSecrets": [],
+            "ingress": {
+              "annotations": {},
+              "enabled": false,
+              "hosts": [],
+              "path": "/",
+              "tls": []
+            },
+            "iq": {
+              "adminPort": 8071,
+              "applicationPort": 8070,
+              "configYaml": {
+                "createSampleData": true,
+                "server": {
+                  "adminConnectors": [
+                    {
+                      "port": 8071,
+                      "type": "http"
+                    }
+                  ],
+                  "applicationConnectors": [
+                    {
+                      "port": 8070,
+                      "type": "http"
+                    }
+                  ]
+                },
+                "sonatypeWork": "/sonatype-work"
+              },
+              "imageName": "registry.connect.redhat.com/sonatype/nexus-iq-server:1.90.0-01-ubi-1",
+              "imagePullPolicy": "IfNotPresent",
+              "imagePullSecret": "",
+              "licenseSecret": "",
+              "memory": "1Gi",
+              "name": "nxiq"
+            },
+            "nameOverride": "",
+            "persistence": {
+              "accessMode": "ReadWriteOnce",
+              "storageSize": "1Gi"
+            },
+            "service": {
+              "annotations": {},
+              "enabled": false,
+              "labels": {},
+              "ports": [
+                {
+                  "name": "nexus-service",
+                  "port": 80,
+                  "targetPort": 80
+                }
+              ]
+            },
+            "serviceAccount": {
+              "create": true,
+              "name": null
+            }
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: "Security"
+    description: |-
+      Nexus IQ Server is a policy engine powered by precise intelligence on open source components.
+    containerImage: registry.connect.redhat.com/sonatype/nxiq-operator-certified
+    repository: https://github.com/sonatype/operator-iq
+    createdAt: 2020-04-24
+    support: Sonatype
+    certified: "true"
+  name: nxiq-operator-certified.v1.90.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: NexusIQ
+      name: nexusiqs.apm.nxiq.com
+      version: v1alpha1
+      description: Nexus IQ Server
+      displayName: NexusIQ
+  description: |-
+    Nexus IQ Server is a policy engine powered by precise intelligence on open source components.
+
+    ## Core Capabilities
+
+    ## Usage
```

Comment:
this whole block looks like a copy & paste of code that's higher up (and maybe even a readme.md file somewhere). Can this be improved? Do we need all of this? Will it be auto-regenerated or will be trying to maintain the duplicates?

Reply:
there are 3 files from this deploy/ directory that get copied down to the bundle/ directory (with slight renaming). they are indeed duplicate. it's the process detailed by the guide.  I don't care for needing to remember to duplicate it either, so I could see a time where I start to automate this duplication.  i'd especially love to be able to build the operator directly from helm charts, but I'm not sure how soon I'll get away with that.